src/test/java/com/flab/bbt/product/service/ProductServiceTest.java

```diff
@@ -0,0 +1,121 @@
+package com.flab.bbt.product.service;
+
+import com.flab.bbt.exception.CustomException;
+import com.flab.bbt.product.domain.Product;
+import com.flab.bbt.product.repository.ProductRepository;
+import com.flab.bbt.product.repository.ProductRepositoryImpl;
+import com.flab.bbt.user.domain.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.doReturn;
+
+@ExtendWith(MockitoExtension.class)
+class ProductServiceTest {
+    private ProductRepository productRepository = new ProductRepositoryImpl();
```

Comment:
오 테스트코드도 작성하셨군요.. 고생 많으셨슴다
근데 Mockito를 추가하셨으면 이 부분은 
```suggestion
    @Mock
    ProductRepository productRepository;
```
로 대체할 수 있지 않나욤 ? ?

Reply:
mock 반영했습니다.